package/boot/uboot-envtools/files/ramips

```diff
@@ -28,6 +28,7 @@ all0256n-8M|\
 all5002)
 	ubootenv_add_uci_config "/dev/mtd1" "0x0" "0x10000" "0x10000"
 	;;
+miwifi-r3|\
```

Comment:
Any reason why you added u-boot env support for this board? As far as I can see, there is nothing which requires write access to the u-boot env.

Reply:
it is need to revert back to stock firmware.
fw_setenv flag_last_success 0